kroma-validator/validator/status.go

```diff
@@ -0,0 +1,12 @@
+package validator
```

Comment:
I think it would be better to locate this file in `kroma/kroma-validator` because it is related to all of kroma-validator components.